draft-connolly-cfrg-xwing-kem.md

```diff
@@ -460,6 +513,26 @@ combiner cannot be assumed to be secure, when used with different
 KEMs. In particular it is not known to be safe to leave
 out the post-quantum ciphertext from the combiner in the general case.
 
+## Binding properties
+Some protocols rely on further properties of the KEM.
+X-Wing satisfies the binding properties MAL-BIND-K-PK and MAL-BIND-K-CT
+(TODO: reference to proof).
+This implies {{KSMW}} X-Wing also satisfies
+
+- MAL-BIND-K,CT-PK
+- MAL-BIND-K,PK-CT
+- LEAK-BIND-K-PK
+- LEAK-BIND-K-CT
+- LEAK-BIND-K,CT-PK
+- LEAK-BIND-K,PK-CT
+- HON-BIND-K-PK
+- HON-BIND-K-CT
+- HON-BIND-K,CT-PK
+- HON-BIND-K,PK-CT
+
+In contrast, ML-KEM on its own does not achieve
+MAL-BIND-K-PK, MAL-BIND-K-CT, nor MAL-BIND-K,PK-CT. {{SCHMIEG}}
```

Comment:
Yeah it looks like they didn't change the algorithms or keygen sufficiently to affect this